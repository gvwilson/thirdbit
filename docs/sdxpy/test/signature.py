def zero():
    print("zero")

def one(value):
    print("one", value)

for func in [zero, one]:
    func()
