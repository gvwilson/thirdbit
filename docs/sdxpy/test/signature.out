zero
Traceback (most recent call last):
  File "/sdx/test/signature.py", line 8, in <module>
    func()
TypeError: one() missing 1 required positional argument: 'value'
